openrpc/src/anchor-platform/schemas/platform_transaction.json

```diff
@@ -0,0 +1,105 @@
+{
+  "platform_transaction": {
```

Comment:
We already have platform schemas defined by SEP because the field values differ between SEPs. Do we want to reuse that? https://github.com/philipliu/stellar-docs/blob/main/openapi/anchor-platform/schemas.yaml#L397-L652

Reply:
Hm, it looks like these work for openrpc. That's fine then.